tests/sparseml/pytorch/image_classification/utils/test_helpers.py

```diff
@@ -0,0 +1,50 @@
+# Copyright (c) 2021 - present / Neuralmagic, Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import shutil
+import tempfile
+
+import pytest
+
+from sparseml.pytorch.image_classification.utils.helpers import save_zoo_directory
+from sparsezoo import Model
+
+
+@pytest.mark.parametrize(
+    "stub",
+    [
+        "zoo:cv/classification/resnet_v1-50/pytorch/sparseml/imagenet/pruned95_quant-none",  # noqa E501
+        "zoo:cv/classification/mobilenet_v1-1.0/pytorch/sparseml/imagenet/pruned_quant-moderate",  # noqa E501
+    ],
+)
+def test_save_zoo_directory(stub):
+    path_to_training_outputs = tempfile.mktemp()
+    save_dir = tempfile.mktemp()
```

Comment:
pytest has a `tmp_dir` fixture that will create & cleanup temporary directories for you!

Reply:
I know, I even used it. I just like to write a test where the reader can see the explicit control over the temp files created, just my quirk. You're probably right tho